src/ansys/hps/client/jms/api/project_api.py

```diff
@@ -278,6 +280,37 @@ def copy_task_definitions(
         """
         return _copy_objects(self.client, self.url, task_definitions, wait=wait)
 
+    def analyze_task_definition(
+        self,
+        task_definition_id: str,
+        evaluator_ids: list[str] = None,
+        scaler_ids: list[str] = None,
+        analytics: bool = True,
+        as_object: bool = True,
+    ) -> AnalyzeResponse:
+        """Compare resource requirements against available compute resources."""
+
+        # we get the task definition as a native dict to more easily translate
+        # the sub-objects into RMS models
```

Comment:
```suggestion
        # Task definition is retrieved as a native dictionar to more easily translate
        # the subobjects into RMS models
```
Avoid "we"--even in comments.